tests/linalg/svd.rs

```diff
@@ -350,6 +361,26 @@ fn svd_fail() {
     assert_relative_eq!(m, recomp, epsilon = 1.0e-5);
 }
 
+#[test]
+#[rustfmt::skip]
+fn svd3_fail() {
+    let m = nalgebra::matrix![
```

Comment:
Might be a good idea to add a comment that this is a real case that actually failed at some point. Otherwise in a few years' time it might look like a very arbitrary test

Reply:
Comment added.